src/components/Nav/Nav.js

```diff
@@ -1,8 +1,90 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './Nav.scss';
 
 const Nav = () => {
-  return <div>김인태</div>;
+  const [serchModal, setSearchModal] = useState(false);
+
+  useEffect(() => {});
+  return (
+    <div className="Nav">
```

Comment:
```suggestion
    <div className="nav">
```
PascalCase는 컴포넌트 네이밍에 쓰입니다.

Reply:
네엥 감사합니다